packages/infra/client/manager/src/actor/setup.rs

```diff
@@ -636,18 +640,22 @@ impl Actor {
 				Result::Ok(cmd_out) => {
 					if !cmd_out.status.success() {
 						tracing::error!(
+							actor_id=?self.actor_id,
 							stdout=%std::str::from_utf8(&cmd_out.stdout)?,
```

Comment:
this can short circuit the function. would be good to do `.unwrap_or("?")` (same as below)